src/core/RecyclerListView.tsx

```diff
@@ -314,6 +316,26 @@ export default class RecyclerListView extends React.Component<RecyclerListViewPr
         //     rowRenderer,
         //     ...props,
         // } = this.props;
+
+        if (!this._derivedContentInsets && this.props.scrollViewProps) {
```

Comment:
Make a utility and move to componentWillReceiveProps.
Render is called multiple times in recyclerview internally.